modules/client/src/internal/client/methods.ts

```diff
@@ -1075,4 +1084,69 @@ export class ClientMethods extends ClientCore implements IClientMethods {
     }
     return version;
   }
+
+  public async isDaoUpdateProposalValid(
+    proposalId: string,
+  ): Promise<DaoUpdateProposalValidity> {
```

Comment:
we need to think about this function to be scaleable for future
like for example it recieve the version number and based on the version it decide if the proposal is valid or not
currectly i don't see a path for future, right? 
or u have some other idea?